internal/cli/cmd/install.go

```diff
@@ -325,6 +325,8 @@ func newCmdUninstallWithHelm() *cobra.Command {
 	}
 
 	addCommonUninstallFlags(cmd, &params)
+
```

Comment:
nit: for consistency with the rest of the file, there shouldn't be a newline here